proto/sentry_protos/snuba/v1/trace_item_filter.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+
+message AndFilter {
+  repeated TraceItemFilter filters = 1;
+}
+
+message OrFilter {
+  repeated TraceItemFilter filters = 1;
+}
+
+message ComparisonFilter {
+  enum Op {
+    OP_UNSPECIFIED = 0;
+    OP_LESS_THAN = 1;
+    OP_GREATER_THAN = 2;
+    OP_LESS_THAN_OR_EQUALS = 3;
+    OP_GREATER_THAN_OR_EQUALS = 4;
+    OP_EQUALS = 5;
+    OP_NOT_EQUALS = 6;
+    OP_LIKE = 7; //string only
+    OP_NOT_LIKE = 8; //string only
+  }
+  AttributeKey key = 1;
+  Op op = 2;
+  AttributeValue value = 3;
+}
+
+message ExistsFilter {
+  AttributeKey key = 1;
+}
+
+// Represents a condition on searching for a particular "trace item"
+// (e.g., spans, replays, errors)
+message TraceItemFilter {
```

Comment:
Do we want to support `NotFilter` to negate the filter condition?

Reply:
Good point, I shall add